pkgs/development/python-modules/mlxtend/default.nix

```
@@ -24,28 +24,48 @@ buildPythonPackage rec {
   };
 
   nativeCheckInputs = [ pytestCheckHook ];
-  # image tests download files over the network
-  pytestFlagsArray = [ "-sv" "--ignore=mlxtend/image" ];
-  # Fixed in master, but failing in release version
-  # see: https://github.com/rasbt/mlxtend/pull/721
-  disabledTests = [ "test_variance_explained_ratio" ];
+  pytestFlagsArray = [
+    "-sv"
+
+    # image tests download files over the network
+    "--ignore=mlxtend/image"
+
+    # tests that import deprecated sklearn datasets
+    "--ignore=mlxtend/evaluate/f_test.py"
+    "--ignore=mlxtend/feature_selection/tests/test_column_selector.py"
+    "--ignore=mlxtend/feature_selection/tests/test_exhaustive_feature_selector.py"
+    "--ignore=mlxtend/feature_selection/tests/test_sequential_feature_selector.py"
+    "--ignore=mlxtend/feature_selection/tests/test_sequential_feature_selector_feature_groups.py"
```

Using `disabledTestPaths` is the modern way ([17.27.1.2.5. Using pytestCheckHook](https://nixos.org/manual/nixpkgs/stable/#user-guide)).

```suggestion
  pytestFlagsArray = [
    "-sv"
  ];
  
  disabledTestPaths = [
    # image tests download files over the network
    "mlxtend/image"
    # tests that import deprecated sklearn datasets
    "mlxtend/evaluate/f_test.py"
    "mlxtend/feature_selection/tests/test_column_selector.py"
    "mlxtend/feature_selection/tests/test_exhaustive_feature_selector.py"
    "mlxtend/feature_selection/tests/test_sequential_feature_selector.py"
    "mlxtend/feature_selection/tests/test_sequential_feature_selector_feature_groups.py"
  ];
```

---

Thanks, updated!